### DISCUSSION TITLE
"allowParrotBreeding"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#73] [Ideas]
Vanilla offers no natural way of Parrot breeding, until now (maybe)!

**Comment by 1024-byteeeee** [comment]
# Today I made a painful and difficult decision

### Changelog
#### https://minecraft-ams.github.io/carpetamsaddition/2024/01/04/v2.24.0/
&emsp;
&emsp;
@1024-byteeeee

**Reply by ghost** [reply]
Thank you!!!
